jmclient/jmclient/wallet_utils.py

```diff
@@ -1224,7 +1224,14 @@ def wallet_gettimelockaddress(wallet, locktime_string):
 
     m = FidelityBondMixin.FIDELITY_BOND_MIXDEPTH
     address_type = FidelityBondMixin.BIP32_TIMELOCK_ID
+
     lock_datetime = datetime.strptime(locktime_string, "%Y-%m")
+    now = datetime.now()
+    if lock_datetime.year < now.year or (
+        lock_datetime.year == now.year and lock_datetime.month <= now.month):
```

Comment:
couldn't you just check if `lock_datetime < now`?

Reply:
Yes, you're right, I can.